homepage-v5/js/script.js

```diff
@@ -133,6 +135,23 @@ function buildDate() {
                       "</font>";
 }
 
+function buildHelp() {
+  var newHelp = "";
+
+  console.log(searchSources[0][0]);
+  console.log(searchSources[0][2]);
+
+  for (var i = 1; i < searchSources.length; i++) {
+    console.log(searchSources[i][0]);
+    console.log(searchSources[i][2]);
+
+    //Will add google logo support when I'm not lazy
```

Comment:
just set your span to the ProductSans font. it's built into the font, which is how I do it.